che-theia-plugins.yaml

```diff
@@ -1,22 +1,30 @@
 version: 1.0.0
 plugins:
+
```

Comment:
@vitaliy-guliy maybe it makes sense to apply some formatter for this file, for example https://jsonformatter.org/yaml-formatter, like it was done for https://github.com/eclipse/che-plugin-registry/blob/master/che-plugins.yaml

Reply:
no problems, I've just added it for easier managing the list of plugins

Reply:
I intentionally added empty lines to separate the plugins each other. Also the formatter removes comments with plugin ID.

The links to the extensions become written in two lines, instead of one line.
```
    extensions:
      - >-
        https://github.com/microsoft/vscode-pull-request-github/releases/download/0.20.0/vscode-pull-request-github-0.20.0.vsix
```
Anything else stay unchanged. Let's leave it as it is at the moment. WDYT?

Reply:
I would prefer it if the comments with the plugin ID are removed, and the `che-theia-plugins.yaml` file stays in the same format as the others `che-*.yaml` files. The `ID` fields are based on the publisher field from the upstream `package.json` files, which can change at any time. Writing the IDs here means that they could be out of date.

Reply:
@ericwill the file is formatted by https://jsonformatter.org/yaml-formatter as @svor mentioed